src/test/java/command/AddDeadlineCommandTest.java

```diff
@@ -0,0 +1,42 @@
+package command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.LinkedList;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import exception.DukeException;
+
+public class AddDeadlineCommandTest {
+    @Test
+    public void testExecute_exceptionThrown() {
+        String[] command = {"deadline", ""};
+        AddDeadlineCommand test = new AddDeadlineCommand(command);
+        try {
+            TaskList list = new TaskList(new LinkedList<>());
+            Ui ui = new Ui();
+            Storage storage = new Storage("data/duke.txt");
+            test.execute(list, ui, storage);
```

Comment:
Do add an Assertions.fail() for cases testing on faulty inputs. 
Reason being test cases will be marked as pass even if there is no exception caught.